src/vivarium/framework/components/parser.py

```diff
@@ -25,11 +25,9 @@
 
 from layered_config_tree import LayeredConfigTree
 
+from vivarium.framework.components.manager import Component, ComponentConfigError
```

Comment:
This isn't importing `Component` from the correct place. It should be
```python
from vivarium import Component
from vivarium.framework.components.manager import ComponentConfigError
```
or if that doesn't work

```python
from vivarium.component import Component
from vivarium.framework.components.manager import ComponentConfigError
```